packages/react-components/react-nav-preview/stories/src/AppItem/AppItemDefault.stories.tsx

```diff
@@ -1,4 +1,4 @@
 import * as React from 'react';
 import { AppItem, AppItemProps } from '@fluentui/react-nav-preview';
 
-export const Default = (props: Partial<AppItemProps>) => <AppItem {...props} />;
+export const Default = (props: Partial<AppItemProps>) => <AppItem />;
```

Comment:
Any reason why this is removed?

```suggestion
export const Default = (props: Partial<AppItemProps>) => <AppItem {...props} />;
```

Reply:
it was crashing stories in the build :-/

Reply:
:o what error were you getting? was it an out of memory error? we're getting a similar error in fai